packages/validator/src/cli-validator/run-validator.js

```diff
@@ -160,10 +157,10 @@ async function runValidator(cliArgs, parseOptions = {}) {
     let originalFile;
     let input;
 
-    if (filesToValidate.length > 1) {
-      logger.info(
-        '\n    ' + chalk.underline(`Validation Results for ${validFile}:`)
-      );
+    if (filesToValidate.length > 1 && context.config.outputFormat !== 'json') {
```

Comment:
I think the `context.config.outputFormat !== 'json'` condition is redundant here, since if both of these conditions were met, we would have rejected by now

Reply:
good catch, I added the other error check after this logic, and forgot to update this